src/components/InstagramFeed.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import './InstagramFeed.css';
+import instaImage from '../img/instagram.svg';
+import Button from './Button';
+
+const buttonInstagram = {
+  title: 'Sledujte nás',
+  type: 'white'
+}
+
+class InstagramFeed extends React.Component {
+  render () {
+    return (
+      <div>
+        <div className='instagramTop'>
+          <h3>Sledujte nás i na instagramu</h3>
```

Comment:
jestli mas naky Insta ucet ktery bych chtela zkusit propojit, muzeme na to kouknout :-) nebo naky fejkovy priprav.